lib/components/tab.vue

```diff
@@ -7,8 +7,7 @@
                    :aria-hidden="localActive ? 'false' : 'true'"
                    :aria-expanded="localActive ? 'true' : 'false'"
                    :aria-lablelledby="controlledBy || null"
-                   v-if="localActive || !lazy"
-                   v-show="localActive || lazy"
+                   v-show="localActive"
```

Comment:
But removing the lazy part will always render with v-show (but just have `display:none;` set), so the content of the tabs will always be in the dom (and potentially active) even though not visible, which may not be the desired state for some users